src/test/java/mission1/step4/domain/car/CarTest.java

```diff
@@ -17,17 +19,17 @@
     @DisplayName("자동차 생성시 이름이 5자를 초과할 경우")
     @Test
     public void createCarByIllegalName() {
-        assertThrows(IllegalArgumentException.class, () -> Car.create("testtest", new MoveByRandomValue()));
+        assertThrows(IllegalArgumentException.class, () -> Car.create("testtest"));
     }
 
     @DisplayName("자동차 전진여부 체크")
     @ParameterizedTest(name="{index} {displayName} 제공값={0} {1} {2}, 결과={3}")
     @CsvSource({"1, 2, 3, 0", "4, 4, 6, 3", "3, 3, 4, 1"})
     public void carProgressCheck(int randomValue1, int randomValue2, int randomValue3, int expectedProgress) {
-        Car car = Car.create("test", new MoveByRandomValue());
-        car.move(randomValue1);
-        car.move(randomValue2);
-        car.move(randomValue3);
+        Car car = Car.create("test");
+        car.move(MoveStrategyCacheForTest.getMoveByRandomValue(randomValue1));
```

Comment:
만들어두신 MoveStrategyProgressForTest 를 활용해보는것도 좋을것 같아요 :)
```suggestion
       car.move(() -> true);
```
람다를 사용해서 표현할수도 있습니다 !